test-data/unit/check-modules.test

```diff
@@ -670,45 +670,45 @@ def f(x: str) -> None: pass
 [case testTypeCheckNamedModule]
 # cmd: mypy -m m.a
 [file m/__init__.py]
-None + 1
```

Comment:
IIRC, some tests are actually run with automatic addition of ``--strict-optional`` flag. Why this changes with your PR?